jsonnet/argokit.libsonnet

```
@@ -1,6 +1,7 @@
+local environment = import '../lib/environment.libsonnet';
 local accessPolicies = import '../lib/accessPolicies.libsonnet';
-
 {
+  Environment: environment,
```

Synes dette hørs mest logisk ut, f.eks. `argokit.Env.variable('FOO', 'bar')`

```suggestion
  Env: environment,
```